src/components/charts/bar.js

```diff
@@ -411,20 +413,58 @@ export default class Bar extends Component {
   renderOptionsDropdown() {
     return (
       <div className="dropdown">
-        <button className="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Configurations
-        <span className="caret"></span></button>
+        <button className="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">
+          Configurations&nbsp;
```

Comment:
What is this change for?

Reply:
This add a space between text and "caret" element.
Now there is no space and looks not good.
More info about this character https://www.w3schools.com/html/html_entities.asp

Reply:
Cool, it makes sense! Thank you for the info! I did not know about it! :D